recipes/py2neo/meta.yaml

```diff
@@ -48,7 +48,7 @@ test:
 about:
   home: http://py2neo.org/
   license: Apache-2.0
-  license_file: LICENSE
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE.txt'
```

Comment:
If this is available in the tarball, you don't need to add to the recipe folder.